src/reducers.js

```diff
@@ -5,12 +5,10 @@
 
 import { combineReducers } from 'redux';
 
-import { reducer as navigation } from './navigation';
-import { reducer as sync } from './sync';
+import { reducer as appReducer } from './navigation';
```

Comment:
This reducer seems to be renamed a lot, going from `navReducer` to `reducer` to `appReducer`. Could we just stick with one? Maybe `navigationReducer`?

Reply:
Resolved with e129dc5